recipes/simbody/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = "3.5.3" %}
+{% set variant = "openblas" %}
+
+package:
+    name: simbody
+    version: {{ version }}
+
+source:
+    fn: Simbody-{{ version }}.tar.gz
+    url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
+    sha256: 8005fbdb16c6475f98e13b8f1423b0e9951c193681c2b0d19ae5b711d7e24ec1
+
+build:
+    number: 200
+    skip: true                # [win]
+    features:
+        - vc14                # [win and py35]
+        - blas_{{ variant }}  # [unix]
+    # NOTE : This is needed to ensure the paths to Simbody's binaries, like
+    # simbody-visualizer, are corrected.
+    detect_binary_files_with_prefix: true
+
+# NOTE: This installs Simbody and the optional visualizer on Linux, Windows, and
+# OSX. We build against the openblas available in conda-forge which includes
+# lapack on Linux and OSX. blas and lapack are not needed on Windows because
+# Simbody includes them. For the visualizer, FreeGLUT is available via
+# anaconda.org and conda-forge but libXi and libXmu are not, so the user will
+# need those to be installed via their Linux package manager. Glut is included
+# with Simbody for Windows and OSX includes a Glut implementation. TODO:
+# Doxygen is the only dependency that is omitted here, so the rendered
+# documentation is not built.
+requirements:
+    build:
+        - toolchain
+        - cmake
+        - blas 1.1 {{ variant }}  # [not win]
+        - openblas 0.2.18*        # [not win]
+        - freeglut                # [linux]
+        - python                  # [win]
+    run:
+        - blas 1.1 {{ variant }}  # [not win]
+        - openblas 0.2.18*        # [not win]
+        - freeglut                # [linux]
+test:
+  requires:
+    - python 3.5.*  # [win and py35]
+  commands:
+    - if not exist %LIBRARY_BIN%\\simbody\\simbody-visualizer exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\Simbody.h exit 1           # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKcommon.h exit 1       # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKlapack.h exit 1       # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKmath.h exit 1         # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKsimbody_aux.h exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKsimbody.h exit 1      # [win]
```

Comment:
On second thought, it should not be different in general. For eg: a user would do,
```
#include <simbody/Simbody.h>
```
in user program. This would work in Unixes, but will not in Windows because Simbody.h is installed in `C:\Program Files\simbody\include\Simbody.h`